crates/rolldown_ecmascript/src/ecma_ast/gen.rs

```diff
@@ -0,0 +1,16 @@
+use oxc::codegen::{self, CodeGenerator, Gen};
+
+pub trait ToSourceString {
```

Comment:
Used for convert any stmt or program to source_string, useful when you debugging.